test/integration/serving/serving_test.go

```diff
@@ -51,7 +51,7 @@ func (kubeControllerManagerTester) StartTestServer(t kubectrlmgrtesting.Logger,
 	if err != nil {
 		return nil, nil, nil, nil, err
 	}
-	return gotResult.Options.SecureServing, gotResult.Config.SecureServing, gotResult.Config.InsecureServing, gotResult.TearDownFn, err
```

Comment:
remove the return param since it's unused now

Reply:
@liggitt We can't do that right now, because `func (kubeControllerManagerTester) StartTestServer` is a implementation of the Interface:
```go
type componentTester interface {
	StartTestServer(t kubectrlmgrtesting.Logger, customFlags []string) (*options.SecureServingOptionsWithLoopback, *server.SecureServingInfo, *server.DeprecatedInsecureServingInfo, func(), error)
}
```

After all the implementations have removed `InsecureServing `, we can change the return param of the Interface.

Reply:
Currently, kube-Scheduler also adopts a similar approach, just return `nil`:
https://github.com/kubernetes/kubernetes/blob/48c8183fc42f2e64ecfde0c9fa8b03f637392a10/test/integration/serving/serving_test.go#L69-L75